docs/users/FAQ.md

```diff
@@ -22,7 +22,13 @@ See SYN [Shop Vision](./SYN Shop Vision.md)
 
 ## What is the best time to visit SYN Shop?
 
-Come on down when ever we're open! Our hours are:
+Unfortunately, due to COVID-19, the SYN Shop is currently closed to non-members.
```

Comment:
maybe we add a link to [synshop covid page](https://synshop.org/covid-19-status)?

Reply:
That's a great idea, but is the lack of a link blocking this PR?  There's no link in the copy it's replacing.

I would like to not have to spend a lot of time in back-and-forth on small copy updates - if it's better than what's up there (which is presently just incorrect), smash the button and open an issue (or a PR).

You'll forgive me if I'm a bit short on patience for your nitpicking of my volunteer PR (now months old) to correct errors which, if merged as-is, would still result in a site that is *just fine* and *less wrong than the one that is up there*.

Reply:
Agreed, not blocking!